charts/urban-os/Chart.lock

```diff
@@ -1,19 +1,19 @@
 dependencies:
 - name: alchemist
   repository: file://../alchemist
-  version: 1.0.0
+  version: 1.0.1
 - name: andi
   repository: file://../andi
-  version: 2.1.3
+  version: 2.2.0
 - name: discovery-api
   repository: file://../discovery-api
-  version: 1.2.3
+  version: 1.3.0
 - name: discovery-streams
   repository: file://../discovery-streams
-  version: 1.0.5
+  version: 1.1.0
 - name: discovery-ui
   repository: file://../discovery-ui
-  version: 1.0.1
```

Comment:
Is this lockfile update intentional?

I can't remember if we were waiting to up versions until alchemist work was complete. Maybe that's a bad memory since the system should still work without alchemist?

This comment is more about reminding me instead of requesting a change.

Reply:
Yes, its just noting the updates to the local charts